src/ZfcUser/Factory/Form/ChangeEmail.php

```diff
@@ -18,13 +18,8 @@ class ChangeEmail implements FactoryInterface
 {
     public function createService(ServiceLocatorInterface $formElementManager)
     {
-        if ($formElementManager instanceof FormElementManager) {
-            $sm = $formElementManager->getServiceLocator();
-            $fem = $formElementManager;
-        } else {
-            $sm = $formElementManager;
-            $fem = $sm->get('FormElementManager');
-        }
+        $fem = $formElementManager;
```

Comment:
I don't understand what you mean with this, the previous code supported that a form element was registered in the ServiceManager instead of the FormElementManager. It is true that the new code only supports form elements registered in the formElementManager.